packages/lesswrong/lib/crud/withUpdate.tsx

```diff
@@ -99,6 +99,7 @@ export const useUpdate = <CollectionName extends CollectionNameString>({ collect
   collectionName: CollectionName,
   fragmentName: FragmentName,
 }): {
+  /** Set a field to `null` to delete it */
```

Comment:
I thought about improving the reciprocal situation[^1], but realized it was already fine. Still, while looking at building it, I added this useful comment.

[^1]: Where you go from a theme set on the EA Forum to testing LW locally, and therefore might want to remove the theme object from your user. (It turns out you don't need to and also it would lead to an error.)